media/js/foundation/annual-report-2019.es6.js

```diff
@@ -0,0 +1,192 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+// Lazyload images
+window.Mozilla.LazyLoad.init();
+
+const modalContainers = document.getElementsByClassName('has-modal');
+const content = document.querySelector('.mzp-u-modal-content');
+const articleArray = document.querySelectorAll('[data-modal-id]');
+
+const modalNextButtonFragment =
+    `<div class="c-modal-next">
+        <button type="button" class="c-modal-button-next" title="Next">
+            Next
+        </button>
+    </div>`;
+
+const modalPrevButtonFragment =
+    `<div class="c-modal-prev">
+        <button type="button" class="c-modal-button-prev" title="Previous">
+            Previous
+        </button>
+    </div>`;
+
+// Set to true when a listener it set, as to not set it multiple times.
+// Resets when the modal is closed.
+let kbInit = null;
+
+function keyboardListenerInit() {
+
+    if (kbInit) {
+        return;
+        // Turn Off
+    }
+
+    document.addEventListener('keyup', keyboardNextPrev, false);
+    kbInit = true;
+}
+
+function keyboardNextPrev(event) {
+    if (!kbInit) {
+        return;
+    }
+
+    switch (event.keyCode) {
```

Comment:
Good question! I think `keyCode` is unlikely to get removed from browsers because it would likely break a lot of existing websites, but we could update this if we think it's worth it. Feel free to file an issue if so?